src/routers/user-router.ts

```diff
@@ -0,0 +1,130 @@
+import { Request, Response, Router } from 'express';
+import { v4 as uuidv4 } from 'uuid';
+
+import asyncHandler from '../handle-midleware/utils';
+import { validateSchema, validateLogin } from '../validation/validation-middleware';
+
+import * as users from '../../data/data.json';
+import { userSchema } from '../validation/schema';
+
+export const router = Router();
+
+export interface User {
+    id: string;
+    login: string;
+    password: string;
+    age: number;
+    isDeleted: boolean;
+}
+
+const getAutoSuggestUsers = (loginSubstring: string, limit: string) =>
```

Comment:
`limit: string` It is strange to get string but after it convert to number

Reply:
changed